### ISSUE TITLE
[Feature Request]: Theme specific images for the integrated toolbar

### ISSUE THREAD

**Issue by PWagner1** [Krypton-Suite/Standard-Toolkit#968]
Now that #872 has been implemented, theme specific images will now need to be applied to them

**Comment by Wagnerp** [comment]
@Smurf-IV Just a thought, could we move the relevant code up to base, then use an image array within the theme bases like how the colours are done?

**Comment by Smurf-IV** [comment]
Yes, that would be sensible, as "All" themes should be able to have an image ;-)
